.github/workflows/yarn-coverage.yml

```diff
@@ -14,6 +14,9 @@ on:
       - openmetadata-ui/src/main/resources/ui/**
 env:
   UI_WORKING_DIRECTORY: openmetadata-ui/src/main/resources/ui
+concurrency: 
+  group: ${{ github.head_ref }}
```

Comment:
We need to use fallback value here as there are multiple events that this CI is listening to.
Ref - https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-a-fallback-value